lib/networks.js

```diff
@@ -75,7 +75,7 @@ function addNetwork(data) {
   });
 
   _.each(_.values(network), function(value) {
-    if (!_.isObject(value)) {
+    if (value && !_.isObject(value)) {
```

Comment:
`if (!_.isUndefined(value) && !_.isObject(value)) {`

to handle the case between `0` and `undefined`